pkg/windows/windows.go

```diff
@@ -431,7 +431,7 @@ func (vm *windows) Bootstrap(desiredVer, apiServerURL, watchNamespace string, cr
 	}
 	// We have to ensure the WICD files exist separately before the rest of the files because we must ensure services
 	// are stopped and their files closed before modifying them.
-	if err := vm.ensureWICDExists(credentials); err != nil {
+	if err := vm.ensureWICDExists(); err != nil {
```

Comment:
I know @sebsoto asked for this change but be aware if we ever decided to revert this PR, this orthogonal change which has nothing to do with this PR will get reverted too.